cpp/include/gherkin/utils.hpp

```diff
@@ -4,6 +4,7 @@
 #include <string>
 #include <string_view>
 #include <regex>
+#include <locale>
```

Comment:
This seems unused?

Reply:
Needed for https://en.cppreference.com/w/cpp/locale/wstring_convert